src/helix-ui/elements/HXFileTileElement.less

```diff
@@ -0,0 +1,35 @@
+@import "./HXElement";
+@import "elements/hx-icon";
+@import (reference) 'components/box';
+@import (reference) "mixins";
+
```

Comment:
Make sure we're applying Helix Link styles instead of browser-default link styles.